Sync `munder` & `mover` (font-size: inherit) UA stylesheet rule as per MathML Spec

Ahmad-S792 opened the pull request:
#### 775f16a37adca5f1c20f3739fd39e8086bc3341d
<pre>
Sync `munder` &amp; `mover` (font-size: inherit) UA stylesheet rule as per MathML Spec

<a href="https://bugs.webkit.org/show_bug.cgi?id=268853">https://bugs.webkit.org/show_bug.cgi?id=268853</a>

Reviewed by Frédéric Wang.

This patch is to align WebKit with Web-Specification [1]:

[1] <a href="https://w3c.github.io/mathml-core/#user-agent-stylesheet">https://w3c.github.io/mathml-core/#user-agent-stylesheet</a>

This patch is to sync UA rule of `font-size` as &apos;inherit&apos; for `munder`
and `mover` with accent as `true` with nth-child of certain depth.

* Source/WebCore/css/mathml.css:
* LayoutTests/TestExpectations: Remove now passing test
* LayoutTests/platform/mac-wk2/imported/w3c/web-platform-tests/mathml/relations/css-styling/scriptlevel-001-expected.txt: Rebaselined
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/mathml/relations/css-styling/scriptlevel-001-expected.txt: Rebaselined
* LayoutTests/platform/ios-wk2/imported/w3c/web-platform-tests/mathml/relations/css-styling/scriptlevel-001-expected.txt: Rebaselined
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/relations/css-styling/scriptlevel-001-expected.txt: Rebaselined

Canonical link: <a href="https://commits.webkit.org/274212@main">https://commits.webkit.org/274212@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b86252740a9c30f2ef9baf193c0f09b2803fae53

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/38181 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/17102 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/40465 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/40731 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/33967 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/40424 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/19938 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/14443 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/32241 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/38754 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/14504 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/33399 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/12587 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/12601 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/34164 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/42009 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/34766 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/34588 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/38397 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/13136 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/10808 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/36587 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/14699 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8619 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/13552 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/14126 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

fred-wang reviewed (approved):
This looks OK, but I don't know if it makes much sense without the full implementation on `math-depth` etc

Ahmad-S792 committed:
Sync `munder` & `mover` (font-size: inherit) UA stylesheet rule as per MathML Spec

https://bugs.webkit.org/show_bug.cgi?id=268853

Reviewed by Frédéric Wang.

This patch is to align WebKit with Web-Specification [1]:

[1] https://w3c.github.io/mathml-core/#user-agent-stylesheet

This patch is to sync UA rule of `font-size` as 'inherit' for `munder`
and `mover` with accent as `true` with nth-child of certain depth.

* Source/WebCore/css/mathml.css:
* LayoutTests/TestExpectations: Remove now passing test
* LayoutTests/platform/mac-wk2/imported/w3c/web-platform-tests/mathml/relations/css-styling/scriptlevel-001-expected.txt: Rebaselined
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/mathml/relations/css-styling/scriptlevel-001-expected.txt: Rebaselined
* LayoutTests/platform/ios-wk2/imported/w3c/web-platform-tests/mathml/relations/css-styling/scriptlevel-001-expected.txt: Rebaselined
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/relations/css-styling/scriptlevel-001-expected.txt: Rebaselined

Canonical link: https://commits.webkit.org/274212@main
